database/seeds/MinecraftServiceTableSeeder.php

```diff
@@ -243,6 +243,41 @@ private function addCoreOptions()
             'startup' => null,
             'script_install' => $script,
         ]);
+
+        $script = <<<'EOF'
+#!/bin/ash
+# Forge Installation Script
+#
+# Server Files: /mnt/server
+apk update
+apk add curl openjdk8
```

Comment:
In order to install the forge files I have to use that to execute `java -jar` in the setup.